File: Codechef/Practice/Chef and Chocolates.cpp
https://www.codechef.com/problems/CCHOCOLATES




#include <iostream>
using namespace std;

int main() {
	// your code goes here
	int t;
	cin >> t;
	while(t--){
	    int x,y,z;
	    cin >> x >> y >> z;
	    cout << (5*x + 10*y) / z << endl;
	}
	return 0;
}

File: Codechef/Practice/Chef and Chocolates.py
https://www.codechef.com/practice/course/basic-math-python/PPYBM01/problems/CCHOCOLATES




t = int(input())

while t > 0:
    x, y, z = map(int, input().split())
    # Your code goes here
    print((5*x + 10*y)//z)
    t -= 1